apps/src/code-studio/pd/professional_learning_landing/selfPacedProgressTable.module.scss

```diff
@@ -17,6 +17,7 @@ table.selfPacedProgressTable {
   height: 0.5rem;
   border-radius: 1rem;
   margin-block-start: 0.25rem;
+  overflow: hidden;
```

Comment:
This fixes a little UI bug on the progress bar if it gets close to the end. I didn't catch this in the original PR.

| Before | After |
| ---- | ---- |
|